tests/binary-expressions/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -352,6 +352,46 @@ x & (y >> z);
 
 `;
 
+exports[`return.js 1`] = `
+function foo() {
+  return this.hasPlugin("dynamicImports") && this.lookahead().type === tt.parenLeft.right;
+}
+
+function foo() {
+  return this.hasPlugin("dynamicImports") && this.lookahead().type === tt.parenLeft.right
+    ? true
+    : false;
+}
+
+function foo() {
+  return this.calculate().compute().first.numberOfThings > this.calculate().compute().last.numberOfThings
+    ? true
+    : false;
+}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+function foo() {
+  return (
+    this.hasPlugin("dynamicImports") &&
+    this.lookahead().type === tt.parenLeft.right
+  );
+}
+
+function foo() {
+  return this.hasPlugin("dynamicImports") &&
+    this.lookahead().type === tt.parenLeft.right
+    ? true
```

Comment:
Should there be another indentation bump here? 🤔

Reply:
@suchipi what do you think?

Reply:
Hmm... I think it would look better in this particular case, but in terms of how the formatting works, it seems like it should be left off.

This is one of those situations where I would probably use parens if I was writing it myself:
```js
return (
  this.hasPlugin("dynamicImports") &&
  this.lookahead().type === tt.parenLeft.right
) ? true
  : whateverIsOnTheNextLine
```

But that's not really consistent with prettier's style, so I think what we have is fine. Maybe we should start a larger discussion about changing the way we use parens in general